app/views/layouts/application.html.erb

```diff
@@ -1,6 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <% if Settings.google_analytics %>
+      <!-- Google tag (gtag.js) -->
+      <script async src="https://www.googletagmanager.com/gtag/js?id=G-L3QLH9N3QH"></script>
```

Comment:
Is the ID safe to hardcode? Or will we only deploy this with `google_analytics: true` in one environment or something?

Reply:
It looks like we only turn this on in prod, so this is fine for now. If we change that, then a follow up PR can address :+1: